### PR TITLE
docs: Add readme for ascii subject

### PR DESCRIPTION
**Pull request by eslopfer**
Add readme for ascii subject.

### CODE REVIEW

**Review by Hamzaelkhatri — APPROVED** [review]
good subject

**Review by zainabdnaya — APPROVED** [review]
good subject

File: subjects/ascii/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+## ascii
+
+### Instructions
+
+Write a function that receives a string and returns a slice with the ASCII values of its characters.
```
[the remark this answers is not in the document]
Comment (eslopfer) [reply subjects/ascii/README.md:5]: Thanks I'll add that

**Comment by eslopfer** [comment]
Rebase and merge as instructed.

**Pull request merged**